### PR TITLE
feat: export graphs as PNG image

### LINKED ISSUE #446: Graph Bitmaps: Create function to export all layers of a graph as a composited bitmap

### PR DESCRIPTION
**Pull request by hoorayimhelping**
Closes #446

Adds `exportImage` to giraffe's exported interface. This takes in an axes canvas reference, and a layer reference, and produces a PNG bitmap of the graphs.

This currently only accepts a single canvas layer. I imagine we'll have to iterate on this.

This also adds stories for taking screenshots of the following graphs:
- line
- stacked line
- histogram
- heatmap
- band chart

### CODE REVIEW

File: giraffe/package.json [hunk 1]
```diff
@@ -105,5 +105,8 @@
   "peerDependencies": {
     "react": "^16.8.0",
     "react-dom": "^16.8.0"
+  },
+  "dependencies": {
+    "merge-images": "^2.0.0"
```
Comment (TCL735) [inline giraffe/package.json:110]: Should this be in giraffe/giraffe/package.json?

**Review by TCL735 — CHANGES_REQUESTED** [review]

File: giraffe/package.json (hunk 1 above)
Comment (hoorayimhelping) [reply giraffe/package.json:110]: yes

Comment (hoorayimhelping) [reply giraffe/package.json:110]: @TCL735 good check but this it actually is the giraffe/giraffe/package.json file. from the root we're in now there's:

```
package.json
   giraffe/package.json
   stories/package.json
```

**Comment by TCL735** [comment]
I just noticed `giraffe/influxdata-giraffe-2.0.0.tgz` is committed as an empty file.

But other than that, lgtm

**Comment by hoorayimhelping** [comment]
>I just noticed giraffe/influxdata-giraffe-2.0.0.tgz is committed as an empty file.

hmmm I was pretty sure I removed that, good eye.

**Review by TCL735 — APPROVED** [review]

**Review by jrenee42 — COMMENTED** [review]
looks good;  if this *just* adds the function then it's good to go.

confirming that adding a button to actually do the export is not included

**Comment by hoorayimhelping** [comment]
>confirming that adding a button to actually do the export is not included

good check! the button is added to the stories I created for this. it's a little janky, but it's fine in a storybook

**Pull request merged**